### PR TITLE
Added Examples to native docs

### PR DESCRIPTION
**Pull request by duckylarps**

### CODE REVIEW

File: ext/native-decls/CancelEvent.md [hunk 1]
```diff
@@ -10,3 +10,9 @@ void CANCEL_EVENT();
 
 Cancels the currently executing event.
 
+## Examples
+```lua
+AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
+            CancelEvent()
```
Comment (4mmonium) [inline ext/native-decls/CancelEvent.md:16]: Hey, maybe I'm nitpicking, but could you please [indent](https://en.wikipedia.org/wiki/Indentation_style) this to the left using `SHIFT + Tab`?

i.e.
```lua
AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
    CancelEvent()
end)
```

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey there, some changes needed. An example (you can adapt) for `SendNUIMessage` can be found [here](https://docs.fivem.net/docs/scripting-reference/runtimes/lua/functions/SendNUIMessage/).

Thank you for your contribution! 🙂

**Comment by Zaps6000** [comment]
> Hey there, some changes needed. An example (you can adapt) for `SendNUIMessage` can be found [here](https://docs.fivem.net/docs/scripting-reference/runtimes/lua/functions/SendNUIMessage/).
> 
> Thank you for your contribution! 🙂

there

File: ext/native-decls/CancelEvent.md [hunk 2]
```diff
@@ -10,3 +10,9 @@ void CANCEL_EVENT();
 
 Cancels the currently executing event.
 
+## Examples
+```lua
+    AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
```
Comment (4mmonium) [inline ext/native-decls/CancelEvent.md:15]: `AddEventHandler` is still incorrectly indented, remove the spaces to the left so it's properly indented. This is how it should look:

```lua
AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
	CancelEvent()
end)
``` 

Code to copy:

`````
```lua
AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
	CancelEvent()
end)
``` 
`````

Maybe just replace your `lua` block with what I just sent you 😀

**Comment by Zaps6000** [comment]
> Full Screen NUI guide for reference

✅

**Comment by Zaps6000** [comment]
> Looks good to me. You could possibly tell users to check out [this](https://docs.fivem.net/docs/scripting-manual/nui-development/full-screen-nui/) Full Screen NUI guide for reference as well; so the user can match up the example you created and further understand what it's doing.
> 
> But so far it looks OK for the user that is only looking for a `lua` implementation example. Thank you for your contribution! 😊

also attempted a c# and js example

File: ext/native-decls/SendNuiMessage.md [hunk 3]
```diff
@@ -13,3 +13,83 @@ BOOL SEND_NUI_MESSAGE(char* jsonString);
...
+    {
+        dynamic data = new
+        {
+            action = "displaySomething",
+            text = "Hello from C#!"
+        };
+
+        string jsonStr = JsonConvert.SerializeObject(data);
+
+        Function.Call((Hash)0x7AE0589093A2E088, jsonStr);
```
Comment (4mmonium) [inline ext/native-decls/SendNuiMessage.md:68]: Please refrain from doing stuff like this on example code 😅

You could use something such as:

```
TriggerEvent("chat:addMessage", new
{
    color = new[] {255, 0, 0},
    multiline = true,
    args = new[] {"Me",  jsonStr}
});
```

More examples [here](https://docs.fivem.net/docs/resources/chat/events/chat-addMessage/)

File: ext/native-decls/SendNuiMessage.md [hunk 4]
```diff
@@ -13,3 +13,83 @@ BOOL SEND_NUI_MESSAGE(char* jsonString);
...
+            text = "Hello from C#!"
+        };
+
+        string jsonStr = JsonConvert.SerializeObject(data);
+
+        Function.Call((Hash)0x7AE0589093A2E088, jsonStr); 
+    }
+}
+```
+> JS Example:
```
Comment (4mmonium) [inline ext/native-decls/SendNuiMessage.md:72]: Remove the `> JS Example:` line, `` ```js `` already displays this.

File: ext/native-decls/SendNuiMessage.md [hunk 5]
```diff
@@ -13,3 +13,83 @@ BOOL SEND_NUI_MESSAGE(char* jsonString);
 * **jsonString**: 
 
 ## Return value
+
+## Examples
+> LUA Example:
```
Comment (4mmonium) [inline ext/native-decls/SendNuiMessage.md:18]: Remove the `> LUA Example:` line, `` ```lua `` already displays this.

File: ext/native-decls/SendNuiMessage.md [hunk 6]
```diff
@@ -13,3 +13,83 @@ BOOL SEND_NUI_MESSAGE(char* jsonString);
...
+
+local result = SendNUIMessage(jsonStr)
+
+if result then
+    print("Message sent to NUI.")
+else
+    print("Failed to send message to NUI.")
+end
+```
+> C# Example:
```
Comment (4mmonium) [inline ext/native-decls/SendNuiMessage.md:35]: Remove the `> C# Example:` line, `` ```cs `` already displays this.

File: ext/native-decls/SendNuiMessage.md [hunk 7]
```diff
@@ -13,3 +13,83 @@ BOOL SEND_NUI_MESSAGE(char* jsonString);
...
+```c#
```
Comment (4mmonium) [inline ext/native-decls/SendNuiMessage.md:36]: This is supposed to be `` ```cs ``, not `` ```c# ``

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey there, the effort is appreciated; requested some changes 😊

**Comment by Zaps6000** [comment]
All right thanks I will update these I didnt know to much about c# so thats why but here let me get those requested changes

**Comment by Zaps6000** [comment]
😬

**Pull request closed without merging**